src/components/GridTemplates/index.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @file Grid templates entry point.
```

Comment:
This goes back to naming, but I wonder if the term `LayoutTemplates` would be more suitable than `GridTemplates`, as the grid is the underlying technology, but the layout is the intended solution.

Reply:
This has been updated 👍